app/services/user/instance_preload_service.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Preloads the instances for given users.
+class User::InstancePreloadService
```

Comment:
Should we test this service?

Reply:
Test added, the failure on travis is random